docusaurus/docs/Flutter/guides/adding_push_notifcations.mdx

```diff
@@ -13,24 +13,223 @@ of new messages and old notifications sometimes need to be updated silently as w
+
+11. After configuring [stream-cli](https://github.com/GetStream/stream-cli) paste the following command on command line using your user ID
+
+```shell
+stream chat:push:test -u <USER-ID> 
+```
+
+You should get a test push notification
+
+### App in the background but still connected
+
+The [StreamChat](https://pub.dev/documentation/stream_chat_flutter/latest/stream_chat_flutter/StreamChat-class.html) widget lets you define a [onBackgroundEventReceived](https://pub.dev/documentation/stream_chat_flutter/latest/stream_chat_flutter/StreamChat/onBackgroundEventReceived.html) handler in order to handle events while the app is in the background, but the client is still connected.
+
+This is useful because it lets you keep the connection alive in cases in which the app goes in the background just for some seconds (eg: multitasking, picking pictures from the gallery...)
+
+You can even customize the [backgroundKeepAlive](https://pub.dev/documentation/stream_chat_flutter/latest/stream_chat_flutter/StreamChat/backgroundKeepAlive.html) duration.
+
+In order to show notifications in such a case we suggest using the package [flutter_local_notifications](https://pub.dev/packages/flutter_local_notifications); follow the package guide to successfully set up the plugin.
+
+Once that's done you should set the [onBackgroundEventReceived](https://pub.dev/documentation/stream_chat_flutter/latest/stream_chat_flutter/StreamChat/onBackgroundEventReceived.html); here is an example:
+
+```dart
+... 
+StreamChat( 
+  client: client, 
+  onBackgroundEventReceived: (e) { 
+    final currentUserId = client.state.user.id; 
+    if (![ 
+          EventType.messageNew, 
+          EventType.notificationMessageNew, 
+        ].contains(event.type) || 
+        event.user.id == currentUserId) { 
+      return; 
+    } 
+    if (event.message == null) return; 
+    final flutterLocalNotificationsPlugin = FlutterLocalNotificationsPlugin(); 
+    final initializationSettingsAndroid = 
+        AndroidInitializationSettings('launch_background'); 
+    final initializationSettingsIOS = IOSInitializationSettings(); 
+    final initializationSettings = InitializationSettings( 
+      android: initializationSettingsAndroid, 
+      iOS: initializationSettingsIOS, 
+    ); 
+    await flutterLocalNotificationsPlugin.initialize(initializationSettings); 
+    await flutterLocalNotificationsPlugin.show( 
+      event.message.id.hashCode, 
+      event.message.user.name, 
+      event.message.text, 
+      NotificationDetails( 
+        android: AndroidNotificationDetails( 
+          'message channel', 
+          'Message channel', 
+          'Channel used for showing messages', 
+          priority: Priority.high, 
+          importance: Importance.high, 
+        ), 
+        iOS: IOSNotificationDetails(), 
+      ), 
+    ); 
+  }, 
+  child: .... 
+); 
+... 
+```
+
+As you can see we §generate a local notification whenever a message.new or notification.message_new event is received.
```

Comment:
Is the '§' doing something or symbolic of something? Remove if a typo.

Reply:
lol that's a typo 
```suggestion
As you can see we generate a local notification whenever a message.new or notification.message_new event is received.
```